language/en/phpbb_ideas_acp.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ *
+ * Ideas extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+if (!defined('IN_PHPBB'))
+{
+	exit;
+}
+
+if (empty($lang) || !is_array($lang))
+{
+	$lang = array();
+}
+
+$lang = array_merge($lang, array(
+	// ACP module
+	'ACP_IDEAS_FORUM_ID'			=> 'Ideas forum',
+	'ACP_IDEAS_FORUM_ID_EXPLAIN'	=> 'Select the forum that ideas topics will be posted to.',
```

Comment:
capital I for Ideas (it's the name of the ext so I think should have capital) ;)